_includes/meta.html

```diff
@@ -18,6 +18,8 @@
     ================================================== -->
   <title>{{ title }}  |  {{site.title}}</title>
   <meta property="og:title" content="{{site.title}}" />
+  <!-- Include DOMPurify for HTML sanitization and security -->
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/dompurify/3.1.7/purify.min.js"></script>
```

Comment:
Does anyone know if there's a way to supply a CDNJS string with the latest version of a package? I'm not a huge fan of how the version needs to be inputted into the src URL to access DOMPurify

Reply:
Is there a reason we're using a CDN for this instead of self-hosting?

Reply:
Using a CDN for this during development makes total sense, but we should self-host this if we elect to use it.

Reply:
@erinysong, leads chatted in refinement, and we're good to see this land in this PR, but can you create an issue to in-source this dependency before this PR is merged? See https://github.com/cisagov/manage.get.gov/issues/2826

Reply:
I think since GIthub's suggestion to sanitize our HTML was called on pre-existing code, this can go in another ticket. I'll uninstall DOMPurify for now and move this to #335 - currently our domain validation prevents users from inputting script characters that can be used in most XSS attacks but we should definitely see if there's a way we can host DOMPurify ourselves

Reply:
@h-m-f-t sorry didn't see your most recent comment until I refreshed  - will take a look at 2826 and update #335 as needed